includes/templates/template_default/templates/tpl_checkout_confirmation_default.php

```diff
@@ -175,7 +175,7 @@
         ?>
 
         <?php
-        echo zen_draw_form('checkout_confirmation', $formActionLink, 'post', 'id="checkout_confirmation" onsubmit="submitonce();"');
+        echo zen_draw_form('checkout_confirmation', $formActionLink, 'post', 'id="checkout_confirmation" onsubmit="submitonce();"' . (defined('MODULE_PAYMENT_PAYPALWPP_CHECKOUTSTYLE') && MODULE_PAYMENT_PAYPALWPP_CHECKOUTSTYLE == 'InContext' && defined('MODULE_PAYMENT_PAYPALWPP_MERCHANTID')    && MODULE_PAYMENT_PAYPALWPP_MERCHANTID    != '' && defined('MODULE_PAYMENT_PAYPALWPP_STATUS')        && MODULE_PAYMENT_PAYPALWPP_STATUS        == 'True' ? ' data-paypal-button="true" data-paypal-id="' . MODULE_PAYMENT_PAYPALWPP_MERCHANTID . '"':''));

```

Comment:
Given that the `payment` class methods all (if the pattern is followed) fire the method in the respective module only if it's defined, there should be no need to touch any of the other modules.